tests/_test_msui/test_mscolab.py

```diff
@@ -334,6 +334,73 @@ def test_view_open(self, qtbot):
         assert tableview.btAddWayPointToFlightTrack.isEnabled()
         assert any(action.text() == "Ins WP" and action.isEnabled() for action in topview.mpl.navbar.actions())
 
+    def test_multiple_views_and_multiple_flightpath(self, qtbot):
+        """
+        checks that we can have multiple topviews with the multiple flightpath dockingwidget
+        and we are able to cycle a login/logout
+        """
+        # more operations for the user
+        for op_name in ["second", "third"]:
+            assert add_operation(op_name, "description")
+            assert add_user_to_operation(path=op_name, emailid=self.userdata[0])
+
+        self._connect_to_mscolab(qtbot)
+        modify_config_file({"MSS_auth": {self.url: self.userdata[0]}})
+        self._login(qtbot, emailid=self.userdata[0], password=self.userdata[2])
+
+        # test after activating operation
+        self._activate_operation_at_index(0)
+        self.window.actionTopView.trigger()
+
+        def assert_active_views():
+            # check 1 view opened
+            assert len(self.window.get_active_views()) == 1
+        qtbot.wait_until(assert_active_views)
+        topview_0 = self.window.listViews.item(0)
+
+        # next topview
+        self.window.actionTopView.trigger()
+        topview_1 = self.window.listViews.item(1)
+
+        def assert_active_views():
+            # check 2 view opened
+            assert len(self.window.get_active_views()) == 2
+        qtbot.wait_until(assert_active_views)
+
+        def select_widget():
+            # open multiple flightpath first window
+            topview_0.window.cbTools.currentIndexChanged.emit(6)
+            assert topview_0.window.docks[5] is not None
+        qtbot.wait_until(select_widget)
+        # activate all operation, this enables them in the docking widget too
+        self._activate_operation_at_index(1)
+        self._activate_operation_at_index(2)
+        self._activate_operation_at_index(0)
+        # ToDo refactor to be able to activate/deactivate by the docking widget and that it can be checked
+
+        def select_widget():
+            # open multiple flightpath second window
+            topview_1.window.cbTools.currentIndexChanged.emit(6)
+            assert topview_1.window.docks[5] is not None
+        qtbot.wait_until(select_widget)
+        # activate all operation, this enables them in the docking widget too
+        self._activate_operation_at_index(1)
+        self._activate_operation_at_index(2)
+        self._activate_operation_at_index(0)
+        # ToDo refactor to be able to activate/deactivate by the docking widget and that it can be checked
+
+        def assert_label_text():
+            # verify logged in
+            assert self.window.usernameLabel.text() == self.userdata[1]
+        qtbot.wait_until(assert_label_text)
+
+        self.window.mscolab.logout()
```

Comment:
Maybe there should be an explicit assert after this to check that the logout was successful.